src/WPML_OptionsManager.php

```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
+            <?php $this->displayMP3Banner() ?>
         </div>
         <?php
     }
 
+    private function displayMP3Banner() {
+        ?>
+        <div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;padding: 20px;position: relative;">
+            <h3><?php _e( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' );?></h3>
+            <?php
+              if (rand(1, 2) === 1) {
+                    $this->displayMP3BannerVersionA();
+              } else {
+                    $this->displayMP3BannerVersionB();
+              }
+            ?>
+        </div>
+        <?php
+    }
+
+    private function displayMP3BannerVersionA() {
+        ?>
+            <ul style="list-style-type:disc">
+                <li><?php _e( '50 email templates to chose from', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Fun email designer', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Automated email marketing', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'WooCommerce emails', 'wp-mail-logging' );?></li>
+                <li><?php _e( '99.1% success email deliverability', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Fast and friendly support', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Over 100,000 active users', 'wp-mail-logging' );?></li>
```

Comment:
Should we add semicolons and a comma at the end of the list items?

Reply:
Nope